tests/test_report_crud.py

```diff
@@ -0,0 +1,115 @@
+import pry
+import json
+from tests.BaseCase import BaseCase
+
+class TestReportCrud(BaseCase):
+    # GET single
+    def test_successful_get_record(self):
+        # Given
+        payload = {
+        "salary": 55000.0,
+        "zipcode": 11111,
+        "credit_score": 695,
+        "monthly_debt": 1100.0,
+        "downpayment_savings": 10000.0,
+        "downpayment_percentage": 10.0,
+        "rent": 1800.0,
+        "goal_principal": 0.0
+        }
+
+        response1 = self.app.post('/api/v1/report', headers={"Content-Type": "application/json"}, data=json.dumps(payload))
+        id = response1.json['data']['id']
+        url = f'/api/v1/report/{id}'
```

Comment:
The GIVEN must first create.

Future iteration of testing: How do we create the object in the database directly instead of using the API?